source/viz.md

```diff
@@ -1457,80 +1544,126 @@ On the other hand, the `maxbins=200` and `maxbins=5` are too small and too big,
+        title="Relative Accuracy (%)"
+    ),
+    y=alt.Y(
+        "count()",
+        stack=False,
+        title="# Measurements"
+    ),
+    color=alt.Color(
+        "Expt:N",  title="Experiment ID"
+    )
+).properties(height=100, width=400)
+
 morley_hist_70 = alt.Chart(morley_rel).mark_bar(opacity=0.6).encode(
-    x = alt.X("relative_accuracy", bin=alt.Bin(maxbins=70), title = "Relative Accuracy (%)"), 
-    y=alt.Y('count()', stack=False, title = "# Measurements"),
-    color = alt.Color("Expt:N",  title = "Experiment ID")).properties(height=100, width=400)
+    x=alt.X(
+        "relative_accuracy",
+        bin=alt.Bin(maxbins=70),
+        title="Relative Accuracy (%)"
+    ),
+    y=alt.Y(
+        "count()",
+        stack=False,
+        title="# Measurements"
+    ),
+    color=alt.Color(
+        "Expt:N",
+        title="Experiment ID"
+    )
+).properties(height=100, width=400)
 
 morley_hist_5 = alt.Chart(morley_rel).mark_bar(opacity=0.6).encode(
-    x = alt.X("relative_accuracy", bin=alt.Bin(maxbins=5), title = "Relative Accuracy (%)"), 
-    y=alt.Y('count()', stack=False, title = "# Measurements"),
-    color = alt.Color("Expt:N",  title = "Experiment ID")).properties(height=100, width=300)
-
-
-
-
-
-final_plot_max_bins = ((morley_hist_default + v_line).facet(row='Expt:N', data=morley_rel, title = "default maxbins") | (morley_hist_200 + v_line).facet(row='Expt:N', data=morley_rel, title = "maxBins=200")) & ((morley_hist_70 + v_line).facet(row='Expt:N', data=morley_rel, title = "maxBins=70") | (morley_hist_5 + v_line).facet(row='Expt:N', data=morley_rel, title = "maxBins=5"))
-
-
-
+    x=alt.X(
+        "relative_accuracy",
+        bin=alt.Bin(maxbins=5),
+        title="Relative Accuracy (%)"
+    ),
+    y=alt.Y(
+        "count()",
+        stack=False,
+        title="# Measurements"
+    ),
+    color=alt.Color(
+        "Expt:N",
+        title="Experiment ID"
+    )
+).properties(height=100, width=300)
+
+final_plot_max_bins = (
+    (morley_hist_default + v_line).facet(row="Expt:N", data=morley_rel, title="default maxbins") |
+    (morley_hist_200 + v_line).facet(row="Expt:N", data=morley_rel, title="maxBins=200")) &
+    ((morley_hist_70 + v_line).facet(row="Expt:N", data=morley_rel, title="maxBins=70") |
+    (morley_hist_5 + v_line).facet(row="Expt:N", data=morley_rel, title="maxBins=5")
+)
 ```
 
 ```{code-cell} ipython3
 :tags: ["remove-cell"]
-glue('final_plot_max_bins', final_plot_max_bins, display=True)
+glue("final_plot_max_bins", final_plot_max_bins, display=True)
 ```
 
 :::{glue:figure} final_plot_max_bins
-:figwidth: 700px 
+:figwidth: 700px
 :name: final_plot_max_bins
 
 Effect of varying number of max bins on histograms.
 :::
 
-#### Adding layers to a `altair` plot object {-}
+#### Adding layers to a `altair` plot object {+}
 
 ```{index} altair; +
 ```
 
-One of the powerful features of `altair` is that you 
+One of the powerful features of `altair` is that you
```

Comment:
OK yeah, regarding my previous point about layering: yes, we should just cover it earlier where vertical line is used first. We can remove this subsection on layering, it won't be necessary once we cover it earlier.